### DISCUSSION TITLE
weird behaviour when using this plugin in shortcuts

### DISCUSSION THREAD

**Discussion by joe23rep** [numToStr/Comment.nvim#240] [Q&A]
its me again. i found out the shortcut but weirdly it does something unexpected.

the shortcut is this:

`keymap("n", "cy", "yy<Plug>(comment_toggle_linewise)<cr>p", opts)`

it basically copies the current line, turns it into a comment and then paste it uncommented again.

this works but weirdly only if theres no line of code directly under it. when there is a line directly under it, it does what it is supposed to do but weirdly it also comments out the line beneath it

maybe you have an idea where this behaviour comes from and if theres a different way i could achieve this shortcut

**Comment by numToStr** [comment]
Can you try it with `<Plug>(comment_toggle_linewise_current)`?
